c/anagrams_kata/anagrams_finder.h

```diff
@@ -0,0 +1,14 @@
+#ifndef ANAGRAMS_FINDER_H
+#define ANAGRAMS_FINDER_H
+
+typedef struct anagrams_finder_s
+{
+    char *wordlist;
+    void (*find)(struct anagrams_finder_s *_this, char *word);
```

Comment:
Are you coming to the Dojo today (22nd august)? I am curious about this member and the underlying decisions. Might be easier to talk in person about this.